src/replication/log_consumer.cpp

```diff
@@ -317,13 +319,13 @@ namespace cubreplication
       }
 
     entry = se;
+    m_ctrl_chn->send_ack (entry->get_stream_entry_end_position ());
```

Comment:
should be stream_entry :: start_position (or at least data_start_position)
At this point, in stream we have only the stream entry header, the contents have not yet reached the slave.